### PR TITLE
Dfr 908 manage bundle event

### PR DESCRIPTION
**Pull request by rishikrsharma**
JIRA link (if applicable)

https://tools.hmcts.net/jira/browse/DFR-908
Change description

[FR 2022]- Contested-Document handling-Manage bundle event

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -1392,5 +1392,40 @@
...
+    "CaseEventID": "FR_manageHearingBundles",
+    "UserRole": "caseworker-divorce-financialremedy-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_manageHearingBundles",
+    "UserRole": "caseworker-divorce-financialremedy-solicitor",
+    "CRUD": "CRU"
```
Comment (gmikerlemos) [inline definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: Shouldn't this be behind a feature flag?

**Review by gmikerlemos — CHANGES_REQUESTED** [review]
I believe we need that block of code in a AuthorisationCaseEvent-manageBundle-nonprod.json

File: definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 1 above)
Comment (rishikrsharma) [reply definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json:1429]: No

**Comment by rishikrsharma** [comment]
> I believe we need that block of code in a AuthorisationCaseEvent-manageBundle-nonprod.json

Not sure about this - never did this before.

**Review by EllisD-B — APPROVED** [review]

**Review by gmikerlemos — APPROVED** [review]

**Pull request merged**